src/tool_settings.rs

```diff
@@ -0,0 +1,30 @@
+use serde::Deserialize;
+use std::fs;
+
+#[derive(Deserialize)]
+pub struct Config {
+    config: ToolsSettings,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct ToolsSettings {
```

Comment:
Nitpick, but I don't love the name of this. It might get better with doc comments: 

```rust
/// ## MyType
///
/// This comment is shown on the documentation website when you run `cargo doc --open`.
/// It uses Markdown, so feel free to be **fancy**!
pub struct MyType;
```